main.gms

```diff
@@ -230,548 +217,1220 @@ $setGlobal c_description  REMIND run with default settings
+***  Setting capacity factors free is numerically expensive but can be helpful to see if negative prices disappear in historic years as the model is allowed to dispatch.
+$setGlobal cm_dispatchSetyDown  off   !! def = off  The amount that te producing any sety can dispatch less (in percent) - so setting "20" in a cm_dispatchSetyDown column in scenario_config will allow the model to reduce the output of this te by 20% 
+*** cm_dispatchSeelDown <- "off", same as cm_dispatchSetyDown but only provides range to capacity factors of electricity generation technologies
+***  cm_steel_secondary_max_share_scenario
+***  defines maximum secondary steel share per region
+***  Share is faded in from cm_startyear or 2020 to the denoted level by region/year.
+***  Example: "2040.EUR 0.6" will cap the share of secondary steel production at 60 % in EUR from 2040 onwards
+$setGlobal cm_dispatchSeelDown  off   !! def = off  The amount that te producing seel can dispatch less (in percent) (overrides cm_dispatchSetyDown for te producing seel)
 
-$setglobal cm_process_based_steel = off; !! def = off
-*** --------------------------------------------------------------------------------------------------------------------------------------------------------------------
-*** --------------------------------------------------------------------------------------------------------------------------------------------------------------------
-***                                  END OF WARNING ZONE
-*** --------------------------------------------------------------------------------------------------------------------------------------------------------------------
-*** --------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+$setglobal cm_secondary_steel_bound  scenario   !! def = scenario
+$setglobal c_GDPpcScen  SSP2EU     !! def = gdp_SSP2   (automatically adjusted by start_run() based on GDPscen) 
+$setglobal cm_demScen  gdp_SSP2EU     !! def = gdp_SSP2EU
+$setglobal c_delayPolicy  SPA0           !! def = SPA0
+$setGlobal cm_emiMktEScoop  off    !! def = off
+$setGlobal cm_emiMktES2020price  30 !! def = 30
+$setGlobal cm_emiMktES2050  off   !! def = off
+$setGlobal c_scaleEmiHistorical  on  !! def = on
+$setGlobal cm_nash_mode  parallel      !! def = parallel
+$SetGlobal cm_quick_mode  off          !! def = off
+$setGLobal cm_debug_preloop  off !! def = off
+$setGlobal c_EARLYRETIRE  on         !! def = on
+*** set conopt version. Warning: conopt4 is in beta
+$setGlobal cm_conoptv  conopt3    !! def = conopt3
+$setGlobal cm_ccsfosall  off        !! def = off
+$setGlobal cm_APscen  SSP2          !! def = SSP2
+$setglobal cm_CES_configuration  indu_subsectors-buil_simple-tran_edge_esm-POP_pop_SSP2EU-GDP_gdp_SSP2EU-En_gdp_SSP2EU-Kap_debt_limit-Reg_62eff8f7   !! this will be changed by start_run()
```

Comment:
Just for aesthetics: Could we write `$setGlobal` everywhere, with consistent capitalization? I know that GAMS allows whatever, but if we use the same capitalization here that we also would use in R, it is one thing less to think about. If wanted, I can write a `sed` command to standardize all `$setGlobal` statements.

Reply:
I agree, but let's open a separate PR after we have merged this one for that

Reply:
Note that a PR which does en-masse transformation will always have the potential for lots of merge conflicts. That's why it would be easier to do this now, when merge conflicts have to be dealt with anyway and no additional merge conflicts will be generated by changing the format en-masse.